guppylang/compiler/cfg_compiler.py

```diff
@@ -130,12 +130,12 @@ def compile_bb(
             branch_port = choose_vars_for_tuple_sum(
                 unit_sum=branch_port,
                 output_vars=[
-                    [v for v in sort_vars(row) if not v.ty.linear]
+                    [v for v in sort_vars(row) if v.ty.droppable]
```

Comment:
drive-by: We should generally output all non-droppable types here instead of just linear ones